### PR TITLE
add newsletter for Aug 22

### PR DESCRIPTION
**Pull request by davidwyld**

### CODE REVIEW

File: _newsletters/2022-08-10-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+### News, Web & Blogs
+* ‘The entire protein universe’: AI predicts shape of nearly every known protein [(Nature)](https://www.nature.com/articles/d41586-022-02083-2) - DeepMind’s AlphaFold tool has determined the structures of around 200 million proteins. 
+* [How to create a Python package in 2022](https://mathspp.com/blog/how-to-create-a-python-package-in-2022) - Useful tutorial/blog summarising the current state of python packaging, using Poetry.
+* [Welcome to Quarto Workshop](https://www.youtube.com/watch?v=yvi5uXQMvu4) - 2hr webcast walking you through [Quarto](https://quarto.org), the evolution of RMarkdown, supporting Python, Julia and much, much more.
+* Code for Thought Podcast
+    * [Code for Thought is going Live](https://software.ac.uk/blog/2022-08-01-code-thought-going-live) - Peter Schmidt reflects on the progress of the Code for Thought podcast and its future development. 
+    * Drones, shrubs and ecology https://tinyurl.com/3yfshcsk Listen to SSI Fellow Isla Myers-Smiths who tells us how they measure climate change in arctic ecosystems and the technical challenges involved.
```
Comment (Robadob) [inline _newsletters/2022-08-10-newsletter.md]: Is this link meant to be like this? (and not wrapping some other text)

File: _newsletters/2022-08-10-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    * Teresa Gomez-Diaz from the University Gustave Eiffel in Paris takes us through open science, open software and the importance of software licensing (often forgotten by engineers). Peter Schmidt was excited to meet Teresa for his first “live” interview https://tinyurl.com/3dwt7k9d
```
Comment (Robadob) [inline _newsletters/2022-08-10-newsletter.md]: Same raw link

File: _newsletters/2022-08-10-newsletter.md (hunk 1 above)
Comment (Robadob) [reply _newsletters/2022-08-10-newsletter.md]: Also shortened links are pretty grim, as they hide the destination.

File: _newsletters/2022-08-10-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+
+### Events
+* [Research Software Hour](https://researchsoftwarehour.github.io/) weekly, Thursdays 20:30 Oslo time / 21:30 Helsinki time - Hosted by members of the Nordic-RSE community, this continues weekly on Twitch. Research Software Hour is an online stream/show about scientific computing and research software. It is designed to provide the skills typically picked up via informal networks; each week, they do some combination of exploring new tools, analyzing and improving someone’s research code, and discussion. Watchers can take part and contribute code to us which they analyze and discuss on stream.
```
Comment (Robadob) [inline _newsletters/2022-08-10-newsletter.md]: Be worth specifying UK time, appears Oslo is UK+1

File: _newsletters/2022-08-10-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    * Teresa Gomez-Diaz from the University Gustave Eiffel in Paris takes us through open science, open software and the importance of software licensing (often forgotten by engineers). Peter Schmidt was excited to meet Teresa for his first “live” interview https://tinyurl.com/3dwt7k9d
+
+
+### Events
+* [Research Software Hour](https://researchsoftwarehour.github.io/) weekly, Thursdays 20:30 Oslo time / 21:30 Helsinki time - Hosted by members of the Nordic-RSE community, this continues weekly on Twitch. Research Software Hour is an online stream/show about scientific computing and research software. It is designed to provide the skills typically picked up via informal networks; each week, they do some combination of exploring new tools, analyzing and improving someone’s research code, and discussion. Watchers can take part and contribute code to us which they analyze and discuss on stream. 
+* [RSECon22](https://rsecon2022.society-rse.org/registration/) 6-8th September - The annual UK conference is hosted in Newcastle this year, **registration for attendees closes 6th July!**
```
Comment (Robadob) [inline _newsletters/2022-08-10-newsletter.md]: Registration has closed

File: _newsletters/2022-08-10-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+* OpenFest 2022 14-15th September - The University library is hosting this conference on the topic of open research practices. [Programme Here](https://docs.google.com/document/d/1_Eo3uOoUv3tcUUK_YUTmWcc-YTszpAQ3JFVHs-C-e9c/edit?usp=sharing) - [Register Here](https://www.eventbrite.co.uk/e/openfest-2022-tickets-383818570677)
+* [GitKon 2022](https://gitkon.com) 11-13th October - GitKraken is hosting an online conference filled with sessions on everything important to Git users.
+* [Computing Insight UK 2022](https://www.scd.stfc.ac.uk/Pages/CIUK-2022-Presentations.aspx) 1-2nd December - A HPC focused conference in Manchester, this year's key theme is Sustainable HPC, abstract submission closes 16th September.
+
+### Opportunities
+#### Funding
+* [Sony Research Award Program](https://www.sony.com/en/SonyInfo/research-award-program) - The Sony Research Award Program provides funding for cutting-edge academic research and helps build a collaborative relationship between faculty and Sony researchers.
```
Comment (Robadob) [inline _newsletters/2022-08-10-newsletter.md]: Applications close Sept 15th

File: _newsletters/2022-08-10-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+* [Write the Docs 2022](https://www.writethedocs.org/conf/prague/2022/) 11-13 September 2022 - Write the Docs brings everyone who writes the docs together in the same room: Programmers, Tech Writers, Support, Designers, Developer Advocates, and more. We all have things to learn from each other, and there’s no better way than sitting together and talking. We invite you to join 300 other folks in our event to explore the art and science of documentation.
+* OpenFest 2022 14-15th September - The University library is hosting this conference on the topic of open research practices. [Programme Here](https://docs.google.com/document/d/1_Eo3uOoUv3tcUUK_YUTmWcc-YTszpAQ3JFVHs-C-e9c/edit?usp=sharing) - [Register Here](https://www.eventbrite.co.uk/e/openfest-2022-tickets-383818570677)
+* [GitKon 2022](https://gitkon.com) 11-13th October - GitKraken is hosting an online conference filled with sessions on everything important to Git users.
+* [Computing Insight UK 2022](https://www.scd.stfc.ac.uk/Pages/CIUK-2022-Presentations.aspx) 1-2nd December - A HPC focused conference in Manchester, this year's key theme is Sustainable HPC, abstract submission closes 16th September.
+
+### Opportunities
```
Comment (Robadob) [inline _newsletters/2022-08-10-newsletter.md]: Late suggestion, we should probably push mention Twins new Tier 2 funding call.
https://rse.shef.ac.uk/collaboration/tier2/
"Get up to 12 days of software engineering support for your HPC/ML/GPU project" (from the email he sent out just now)

Comment (davidwyld) [reply _newsletters/2022-08-10-newsletter.md]: agreed.

**Pull request merged**